registry/accessing-the-registry.adoc

```diff
@@ -49,8 +49,10 @@ include::modules/registry-viewing-logs.adoc[leveloffset=+1]
 
 include::modules/registry-accessing-metrics.adoc[leveloffset=+1]
 
-.Additional resources
+include::modules/
 
+== Additional resources
```

Comment:
Please add a unique anchor above this heading. You could use a combination of the anchor ID of the assembly heading and `-additional-resources`, as in the following example:

`[id="accessing-the-registry-additional-resources"]`

Contributors can subsequently cross-reference this section by using the unique anchor ID.